test/integration/tests/collateralizationOracle.ts

```diff
@@ -162,7 +162,15 @@ describe('e2e-collateralization', function () {
 
         const numTokens = (await collateralizationOracle.getDepositsForToken(element)).length;
         doLogging && console.log(`Address count for token ${addresses[i]}: ${numTokens}`);
-        expect(numTokens).to.be.equal(collateralizationAddresses[addresses[i]].length);
+        expect(numTokens).to.be.equal(
```

Comment:
what is this?

Reply:
I improved the print when this contract fails because if the collateralizationOracle protocol config file is not properly filled, this e2e tests without a useful message otherwise